lib/building.rb

```diff
@@ -0,0 +1,45 @@
+class Building
+  attr_reader :units,
+              :renters,
+              :rented_units
+
+  def initialize
+    @units = []
+    @renters = []
+    @rented_units = []
```

Comment:
These are not attributes of the building. They shouldc be determined from the units.